src/main/webapp/app/shared/orion/orion.ts

```diff
@@ -16,6 +16,7 @@ export interface OrionState {
  */
 export enum ExerciseView {
     STUDENT = 'STUDENT',
+    TUTOR = 'TUTOR',
```

Comment:
Maybe adding the Editor as well would be a good idea, even if it is not necessary for this particular use case

Reply:
This naming is confusing, but the exercise view refers to the ways an exercise can be opened in Orion, which just happens to be called similarly to the roles in Artemis - it is not possible to open an exercise from an editor's perspective - if the editor participates, its student, if they edit the exercise, its instructor, if they assess, its tutor